dis_snek/client/client.py

```diff
@@ -1543,7 +1548,10 @@ def shed_scale(self, scale_name: str, **unload_kwargs) -> None:
 
         """
         if scale := self.get_scale(scale_name):
-            return self.unload_extension(inspect.getmodule(scale).__name__, **unload_kwargs)
+            self.unload_extension(inspect.getmodule(scale).__name__, **unload_kwargs)
+
+            if self.sync_scales:
+                asyncio.create_task(self.synchronise_interactions())
```

Comment:
missing a return here?

Reply:
Won't it raise ScaleLoadExecption below since that is outside of the if condition

Reply:
Resolved in 14b75c93f198b1996bb0e41996d88340cba53981

didnt add a return, instead used an else. same effect though